src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +178,119 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
+                    return Localization.lang("A multi-volume book.");
+                }
+                case MvCollection -> {
+                    return Localization.lang("A multi-volume collection.");
+                }
+                case MvProceedings -> {
+                    return Localization.lang("A multi-volume proceedings entry.");
+                }
+                case MvReference -> {
+                    return Localization.lang("A multi-volume reference entry. The standard styles will treat this entry type as an alias for mvcollection.");
+                }
+                case Online -> {
+                    return Localization.lang("This entry type is intended for sources such as web sites which are intrinsically online resources.");
+                }
+                case Reference -> {
+                    return Localization.lang("A single-volume work of reference such as an encyclopedia or a dictionary.");
+                }
+                case Report -> {
+                    return Localization.lang("A technical report, research report, or white paper published by a university or some other institution.");
+                }
+                case Set -> {
+                    return Localization.lang("An entry set is a group of entries which are cited as a single reference and listed as a single item in the bibliography.");
+                }
+                case SuppBook -> {
+                    return Localization.lang("Supplemental material in a book. This type is provided for elements such as prefaces, introductions, forewords, afterwords, etc. which often have a generic title only.");
+                }
+                case SuppCollection -> {
+                    return Localization.lang("Supplemental material in a collection.");
+                }
+                case SuppPeriodical -> {
+                    return Localization.lang("Supplemental material in a periodical. This type may be useful when referring to items such as regular columns, obituaries, letters to the editor, etc. which only have a generic title.");
+                }
+                case Thesis -> {
+                    return Localization.lang("A thesis written for an educational institution to satisfy the requirements for a degree.");
+                }
+                case WWW -> {
+                    return Localization.lang("An alias for online, provided for jurabib compatibility.");
+                }
+                case Software -> {
+                    return Localization.lang("Computer software. The standard styles will treat this entry type as an alias for misc.");
+                }
+                case DATESET -> {
+                    return Localization.lang("A data set or a similar collection of (mostly) raw data.");
+                }
+                default -> {
+                    return "";
+                }
+            }
+        } catch (Exception e) {
+            return "";
```

Comment:
Why do you use a try catch block? With the switch case default you have dealt with all cases and I don't see anything tha throws an exception? (Although it's not needed here, just as a tip for the future: Never catch the generic "Exception" type, always catch the most specific exceptions, see "  (see Item 58 of Joshua Bloch's Effective Java)
"

Reply:
When i cast entryType into StandardEntryType and it does not belong in the Standard Entry Types it throws an exception, e.g. it belongs to IEEETran. So I use a try catch block in order to avoid that.

Reply:
Ah I see, well, then do a check  before with instanceof StandardEntryType
And as of jdk 14 you can use the new pattern matching syntax
https://www.baeldung.com/java-pattern-matching-instanceof

Reply:
@Siedlerchr ok I will do and thanks for your review!